tests/unit/h/views/account_signup_test.py

```diff
@@ -461,6 +510,11 @@ def test_get(
 
         response = views.get()
 
+        jwt_service.decode_symmetric.assert_called_once_with(
+            sentinel.idinfo,
+            audience=JWTAudience.SIGNUP_ORCID,
+            payload_class=IDInfo,
+        )
```

Comment:
Noticed that this assertion was missing so added it here. The code that makes this call already exists on `main`.